src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_params.py

```diff
@@ -265,7 +265,8 @@ def load_arguments(self, _):
         VMPriorityTypes = self.get_models('VirtualMachinePriorityTypes', resource_type=ResourceType.MGMT_COMPUTE)
         c.argument('name', name_arg_type)
         c.argument('nat_backend_port', default=None, help='Backend port to open with NAT rules.  Defaults to 22 on Linux and 3389 on Windows.')
-        c.argument('single_placement_group', default=None, help="Enable single placement group. This flag will default to True if instance count <=100, and default to False for instance count >100.", arg_type=get_enum_type(['true', 'false']))
+        c.argument('single_placement_group', default=None, help="Enable replicate using fault domains within the same cluster", arg_type=get_enum_type(['true', 'false']))
+        c.argument('platform_fault_domain_count', default=None, type=int, help='Fault Domain count for each placement group in the availability zone', min_api='2017-12-01')
```

Comment:
Why are you specifying `default=None`?  VMSS create is a custom command, so that should be established in its signature.